packages/webapp/src/inductions/components/induction-journeys/inviter-witness-journey.tsx

```diff
@@ -0,0 +1,193 @@
+            <Link onClick={() => setIsReviewingVideo(true)}>click here</Link>.
+            Keep in mind that modifying the induction video will reset any
+            endorsements.
+        </Text>
+    </div>
+);
+
+interface Props {
+    endorsements: Endorsement[];
+    induction: Induction;
+    inductionStatus: InductionStatus;
+}
+
+export const InviterWitnessJourney = ({
+    endorsements,
+    induction,
+    inductionStatus,
+}: Props) => {
+    const [ualAccount] = useUALAccount();
+    const [submittedVideo, setSubmittedVideo] = useState(false);
+    const [isReviewingVideo, setIsReviewingVideo] = useState(false);
+    const { data: isCommunityActive } = useIsCommunityActive();
+
+    const memberData = convertPendingProfileToMemberData(induction);
+
+    // Inviter video submission confirmation
+    if (submittedVideo) {
+        // not possible in Genesis mode
+        return (
+            <Container
+                step={InductionStepInviter.VideoAndEndorse}
+                memberPreview={memberData}
+            >
+                <InductionVideoSubmitConfirmation />
+            </Container>
+        );
+    }
+
+    const renderVideoStep = () => (
+        // not possible in Genesis mode
+        <Container
+            step={InductionStepInviter.VideoAndEndorse}
+            memberPreview={memberData}
+        >
+            <InductionVideoFormContainer
+                induction={induction}
+                isReviewingVideo={isReviewingVideo}
+                setSubmittedVideo={setSubmittedVideo}
+            />
+        </Container>
+    );
+
+    if (isReviewingVideo) {
+        return renderVideoStep();
+    }
+
+    switch (inductionStatus) {
+        case InductionStatus.PendingProfile:
+            return (
+                <Container
+                    step={
+                        isCommunityActive
+                            ? InductionStepInviter.PendingProfile
+                            : InductionStepGenesis.Profile
+                    }
+                >
+                    <WaitingForProfile induction={induction} />
+                </Container>
+            );
+        case InductionStatus.PendingCeremonyVideo:
+            return renderVideoStep();
+        case InductionStatus.PendingEndorsement: // not possible in Genesis mode
+            const userEndorsementIsPending =
+                endorsements.find((e) => e.endorser === ualAccount?.accountName)
+                    ?.endorsed === 0;
+
+            return (
+                <Container
+                    step={InductionStepInviter.VideoAndEndorse}
+                    memberPreview={memberData}
+                >
+                    <Heading size={1} className="mb-2">
+                        Endorsements
+                    </Heading>
+                    <InductionExpiresIn induction={induction} />
+                    <EndorsementsStatus endorsements={endorsements} />
+                    {userEndorsementIsPending ? (
```

Comment:
I think that we should display the recommend review even if the endorser has not endorsed yet. Actually I think it's the correct behavior, in case the endorser has found an issue in the video and want's to fix it before endorsing it. He would need to endorse it and quickly click in review video (it could be worse if he is the last endorser).

Reply:
A recommendation and link to review the video is already inside the `<InductionEndorsementForm />` component here. So it appears in either case already.

I will check, however, to see if that component can also just use `<RecommendReview />`. There might be some differences in wording that makes the one inside `<InductionEndorsementForm />` special.

Reply:
Awwww okay, got it! you are absolutely right, I just re-checked it, and I missed it!

Reply:
Yeah, I went ahead and consolidated those.